### PR TITLE
add option to specify post-upgrade argument in make-upgrade-canister-proposal

### PR DESCRIPTION
**Pull request by mraszyk**

### CODE REVIEW

**Review by lwshang — APPROVED** [review]

File: tests/outputs/make-upgrade-canister-proposal.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ Sending message with
           UpgradeSnsControlledCanister = record {
             new_canister_wasm = blob "\00asm\01\00\00\00";
             canister_id = opt principal "pycv5-3jbbb-ccccc-ddddd-cai";
+            canister_upgrade_arg = null;
```
Comment (ghost) [inline tests/outputs/make-upgrade-canister-proposal.txt:18]: The only thing I think that was missing from this PR is a test where this parameter was not null. @mraszyk @lwshang
